dev/bots/utils.dart

```diff
@@ -260,3 +300,297 @@ Future<bool> _isPortAvailable(int port) async {
 String locationInFile(ResolvedUnitResult unit, AstNode node, String workingDirectory) {
   return '${path.relative(path.relative(unit.path, from: workingDirectory))}:${unit.lineInfo.getLocation(node.offset).lineNumber}';
 }
+
+String? _shuffleSeed;
+
+String get shuffleSeed {
+  if (_shuffleSeed == null) {
+    // Attempt to load from the command-line argument
```

Comment:
this is actually an env var now

Reply:
Is that a recent change? I can update with this PR as well, but was originally intending to keep existing functionality intact with this refactor

Reply:
I updated to first check existing value of shuffleSeed, and otherwise fall back on the env var